app/containers/Search/saga.js

```diff
@@ -17,13 +17,16 @@ export function* inputChange(action) {
   const input = action.payload;
 
   try {
+    yield put(typeAheadLoading(true));
```

Comment:
Why not handle this in the reducer?

Reply:
Do you mean adding a typeAheadStartLoading/typeAheadStopLoading actions instead?
In that case, this approach is just less code and does the same thing.

Reply:
@janjaap The loading state changes are now simplified and done in the reducer.